docs/development/plugins/getting-started.md

```diff
@@ -32,12 +32,23 @@ Before starting plugin development, ensure you have:
 
 ## Quick Start: Your First Plugin
 
-Let's create a simple plugin that displays "Hello Headlamp!" in the top navigation bar.
+Let's create a simple plugin that displays "Hello Headlamp!" in the top navigation bar. Before we start, it's important to note that plugins can be created and developed outside of the Headlamp repository. This allows for greater flexibility and ease of management.
+
+Please be sure to check that the required plugin development dependencies are correctly installed.
+
+```bash
+# Navigate to the headlamp-plugin directory within the Headlamp repository
+cd headlamp/plugins/headlamp-plugin
+
+# Install dependencies
+npm install
+```
 
 ### Step 1: Create the Plugin
 
-Run the following command to scaffold a new plugin using the headlamp-plugin
-tool. This command should be run where your plugin project (code) will live, typically in a dedicated projects directory (do not run this in Headlamp's
+With dependencies installed and your preferred instance of Headlamp running, navigate to where you would like to create your plugin. In your preferred location, run the following command to scaffold a new plugin using the headlamp-plugin tool.
```

Comment:
I don't think this change is necessary if we are not telling users to run npm install inside headlamp-plugin.